site/en/docs/extensions/reference/debugger/index.md

```diff
@@ -3,11 +3,7 @@ api: debugger
 has_warning: This permission <a href="/docs/extensions/mv3/permission_warnings/#permissions_with_warnings">triggers a warning</a>.
 ---
 
-## Notes
-
-As of today, attaching to the tab by means of the debugger API and using embedded Chrome DevTools
-with that tab are mutually exclusive. If user invokes Chrome DevTools while extension is attached to
-the tab, debugging session is terminated. Extension can re-establish it later.
+## Note
```

Comment:
@dotproto, it seems like we should just remove 'as of today' and start the note at 'Attaching'. What do you think?

Reply:
Ping!

Reply:
As I recall, Chrome now supports having multiple debuggers attached simultaneously, but I don't know when multi-debugger support landed. As such, I think it's best to remove the lines as originally suggested in this PR.

Reply:
@dotproto Is it [this update](https://developer.chrome.com/blog/new-in-devtools-63/#multi-client) from M63?